File: interfaces/lcm-0.2.1/examples/matlab/sendmessage.m
lc = lcm.lcm.LCM.getSingleton();

msg = lcmtypes.example_t();

msg.timestamp = 0;
msg.position = [1  2  3];
msg.orientation = [1 0 0 0];
msg.ranges = 1:15;
msg.num_ranges = length(msg.ranges);

lc.publish('EXAMPLE', msg);

File: interfaces/lcm-0.3.1/examples/python/send-message.py
import lcm
import time

from example_t import example_t

lc = lcm.LCM()

msg = example_t()
msg.timestamp = int(time.time() * 1000000)
msg.position = (1, 2, 3)
msg.orientation = (1, 0, 0, 0)
msg.ranges = range(15)
msg.num_ranges = len(msg.ranges)

lc.publish("EXAMPLE", msg.encode())
